tests/settings-rest-controller-test.php

```
@@ -0,0 +1,281 @@
+if (!function_exists('__')) {
+    function __($text) { return $text; }
+}
+if (!function_exists('register_rest_route')) {
+    function register_rest_route($namespace, $route, $args) {
+        // Capture registered routes for testing
+        global $registered_routes;
+        $registered_routes[] = ['namespace' => $namespace, 'route' => $route, 'args' => $args];
+    }
+}
+if (!function_exists('current_user_can')) {
+    function current_user_can($cap) { return true; }
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$cap'. (undefined)

(UnusedFormalParameter)